.github/workflows/e2e-suite.yml

```diff
@@ -95,7 +105,14 @@ jobs:
         run: |
           timestamp=$(date +'%Y%m%d%H%M')
           report_filename="${timestamp}_cli_test_report.xml"
-          make testint TEST_ARGS="--junitxml=${report_filename}" MODULE="${{ inputs.module }}"
+          echo "Running tests, report: ${report_filename}"
+          if [ -n "$OPENAPI_SPEC_URL" ]; then
+            echo "Running tests with OpenAPI spec from: $OPENAPI_SPEC_URL"
+            make testint TEST_ARGS="--junitxml=${report_filename}"
```

Comment:
Looks like we are running the same command, can we just leave this as it is? 
MODULE parameter was cut out as well